dart/lib/src/protocol/sentry_culture.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:meta/meta.dart';
+
+/// Culture Context describes certain properties of the culture in which the
+/// software is used.
+@immutable
+class SentryCulture {
+  static const type = 'culture';
+
+  const SentryCulture({
+    this.calendar,
+    this.displayName,
+    this.locale,
+    this.is24HourFormat,
+    this.timezone,
+  });
+
+  factory SentryCulture.fromJson(Map<String, dynamic> data) => SentryCulture(
+        calendar: data['calendar'],
+        displayName: data['display_name'],
+        locale: data['locale'],
+        is24HourFormat: data['is_24_hour_format'],
+        timezone: data['timezone'],
+      );
+
+  /// Optional: For example `GregorianCalendar`. Free form string.
+  final String? calendar;
+
+  /// Optional: Human readable name of the culture.
+  /// For example `English (United States)`
+  final String? displayName;
+
+  /// Optional. The name identifier, usually following the RFC 4646.
+  /// For example `en-US` or `pt-BR`.
+  final String? locale;
+
+  /// Optional. boolean, either true or false.
+  final bool? is24HourFormat;
+
+  /// Optional. The timezone of the locale. For example, `Europe/Vienna`.
+  final String? timezone;
+
+  /// Produces a [Map] that can be serialized to JSON.
+  Map<String, dynamic> toJson() {
+    return <String, String>{};
```

Comment:
does that work? we should populate the map right

Reply:
Oh shoot, yeah you're right.